### DISCUSSION TITLE
Initialization issue

### DISCUSSION THREAD

**Discussion by SvenRuin** [Networking-for-Arduino/EthernetENC#67] [Q&A] [answered]
I'm using Arduino UNO R4 Minima with Ethernet Shield. My application program first included:
#include <Ethernet.h>
EthernetClient cli = 0;
The reason is that I'm using Arduino Telnet Server code based on https://gist.github.com/atomsfat/1813823. After changing the library to EthernetENC (#include <EthernetENC.h>), the program did not compile. Compilation error: conversion from 'int' to 'EthernetClient' is ambiguous. I tried the following change:
EthernetClient cli;
Then the program compiles. However, it seems to crash on the Arduino. I don't know yet if it is related to the above change. If it is, does anyone have advice how to solve the issue? Can EthernetENC.h not be used as a replacement for Ethernet.h this way?

**Comment by JAndrassy** [comment]
what Ethernet shield?

**Reply by SvenRuin** [reply]
Ethernet2 (with chip W5500)

**Reply by JAndrassy** [reply, marked as answer]
but this library is for the ENC28j60 ethernet chip

**Reply by SvenRuin** [reply]
Thank you Juraj. As I now understand, EthernetENC is not compatible with my shield.
